iota-checks.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+          "http://checkstyle.sourceforge.net/dtds/configuration_1_3.dtd">
+<!--
+
+IOTA checkstyle draft
+
+-->
+<module name="Checker">
+    <!--
+        If you set the basedir property below, then all reported file
+        names will be relative to the specified directory. See
+        http://checkstyle.sourceforge.net/5.x/config.html#Checker
+
+        <property name="basedir" value="${basedir}"/>
+    -->
+    <property name="fileExtensions" value="java, properties, xml"/>
+    <module name="TreeWalker">
+        <module name="NeedBraces"/>
```

Comment:
We should also use `<module name="AvoidInlineConditionals"/>`

Reply:
@th0br0 I can't naively make this change without producing checkstyle errors.
I opened GalRogozinski#3 to make sure this change will happen.